content/blog/2024/12/end-year-2024.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "The Year in Review: Kubewarden's Progress in 2024"
+authors:
+  - Martin Kravec
+  - John Krug
+  - José Guilherme Vanz
+date: 2024-12-19
+---
+
+It was an exciting year for Kubewarden policy management. We had new features,
+performance improvements, and have been working towards a regular release
+schedule.
+
+The year has seen work in these areas:
+
+- performance and reliability 
+- scalability improvements to reduce complexity and improve security 
+- adding CEL policies and policy grouping using logical operators 
+- improving community outreach
+
+Kubewarden
+[1.10](https://www.kubewarden.io/blog/2024/01/kubewarden-1-10-release/) has
+optimizations for policy server performance. Memory usage was improved,
+enabling constant consumption even in large deployments. Policy evaluation now
+operates on-demand, consuming resources only as needed.
+
+Policy authors can now create context-aware WASI policies with Kubernetes
+resource insights. This expands Kubewarden’s flexibility policies to use
+Kubernetes capabilities.
+
+Kubewarden
+[1.11](https://www.kubewarden.io/blog/2024/03/kubewarden-1-11-release/) brought
+audit scanner improvements. These included parallelized operations, reduced
+resource consumption, and improved efficiency. Furthermore, new host
+capabilities allowed policies to [retrieve OCI image
+manifests](https://www.kubewarden.io/blog/2024/03/oci-manifest-capability/),
+helping developers enforce security standards by inspecting container
+configurations in detail.
+
+Releases
+[1.12](https://www.kubewarden.io/blog/2024/04/kubewarden-1-12-release/) and
+[1.13](https://www.kubewarden.io/blog/2024/06/kubewarden-1-13-release/) were
+about high availability and resource management. PolicyServer deployments now
+have support for PodDisruptionBudgets, affinity rules, and resource limits.
+This improves stability in production environments. Gatekeeper policies saw a
+55% performance boost. Memory usage was reduced for clusters managing thousands
+of resources.
+
+The introduction of [CEL
+policies](https://www.kubewarden.io/blog/2024/06/welcome-cel-policy/) in
+release
+[1.14](https://www.kubewarden.io/blog/2024/06/kubewarden-1-14-release/),
+enabled the use of the Common Expression Language for policies. This is
+backward-compatible with Kubernetes’ ValidatingAdmissionPolicies.
```

Comment:
We are not backward-compatible but just compatible. Maybe we could use something like "a drop-in replacement "

Reply:
Done